Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -25,6 +25,7 @@ class BrowserViewController: UIViewController {
     private var searchController: SearchViewController?
     private var webViewContainer: UIView!
     private let uriFixup = URIFixup()
+    private var screenshotHelper: ScreenshotHelper!
```

Comment:
Since this isn't a constant, I would probably just make it optional. Unwrapping it here just keeps the compiler from helping us.